cps/templates/user_edit.html

```diff
@@ -80,8 +80,8 @@ <h1>{{title}}</h1>
           <label for="Show_detail_random">{{_('Show Random Books')}}</label>
       </div>
       {% if ( g.user and g.user.role_admin() and not new_user ) %}
-      <a href="#" id="get_user_tags" class="btn btn-default" data-toggle="modal" data-target="#restrictModal">{{_('Add allowed/denied Tags')}}</a>
-      <a href="#" id="get_user_column_values" class="btn btn-default" data-toggle="modal" data-target="#restrictModal">{{_('Add allowed/denied custom column values')}}</a>
+      <a href="#" id="get_user_tags" class="btn btn-default" data-toggle="modal" data-target="#restrictModal">{{_('Edit Tag Restrictions')}}</a>
+      <a href="#" id="get_user_column_values" class="btn btn-default" data-toggle="modal" data-target="#restrictModal">{{_('Edit Custom Column Restrictions')}}</a>
```

Comment:
See my comments on the other "View Restrictions" texts. I'm fine with Edit instead of add, you are right this is what it does. Would "Edit Tag Visibilities" something plausible? I have no good idea for the Custom Column one